<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QMainWindow" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QSpinBox,
QDoubleSpinBox {
	margin-left:25px;
	margin-right:25px;
}

QSpinBox::up-button,
QDoubleSpinBox::up-button  {
	subcontrol-origin: margin;
	subcontrol-position: center right;
	background-color: #FFFFFF;
	image: url(plus.png);
	left: 1px;
	height: 80px;
	width: 80px;
}

QSpinBox::down-button,
QDoubleSpinBox::down-button  {
	subcontrol-origin: margin;
	subcontrol-position: center left;
	background-color: #FFFFFF;
	image: url(minus.png);
	right: 1px;
	height: 80px;
	width: 80px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>31</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Automatic Mode Settings</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>221</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.SF NS Text</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Respiratory Rate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>381</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.SF NS Text</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Insp/Expir Ratio (I:E)   1  /</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>120</y>
        <width>781</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_start_auto">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>220</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_load_preset_auto">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>220</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Load Presets</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBox_rr">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>511</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBox_insp_expir_ratio">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>411</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_close_1">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>220</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>Assisted Mode Settings</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_start_assist">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>220</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_load_preset_assist">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>220</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Load Presets</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pressure trigger</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Flow trigger (lpm)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>251</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Minimal Respiratory Rate</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBox_pressure_trigger">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>501</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBox_flow_trigger">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>501</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBox_min_resp_rate">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>240</y>
        <width>321</width>
        <height>80</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_enable_backup">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>201</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enable Backup</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>781</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>781</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>240</y>
        <width>20</width>
        <height>71</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_close_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>220</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
